src/main/java/org/jabref/logic/search/SearchQuery.java

```diff
@@ -133,7 +164,21 @@ public boolean isRegularExpression() {
         // compile the words to a regular expression in the form (w1)|(w2)|(w3)
         StringJoiner joiner = new StringJoiner(")|(", "(", ")");
         for (String word : words) {
-            joiner.add(regularExpression ? word : Pattern.quote(word));
+            if (regularExpression) {
+                joiner.add(word);
+            }
+            else {
```

Comment:
Isn't checkstyle complaining about putting this in two lines? Has no effect, but looks somewhat odd...

Reply:
Yeah, it's a mistake, I'll change it. Also looks odd to me. Checkstyle wasn't complaining.